pysoa/common/serializer/msgpack_serializer.py

```diff
@@ -67,6 +68,7 @@ class MsgpackSerializer(BaseSerializer):
     EXT_CURRINT = 2
     EXT_DATE = 3
     EXT_DATETIME = 1
+    EXT_DATETIME_UTC = 10
```

Comment:
`10` looks a deliberately chosen, when the next logical id would be `6`, why?

Reply:
@nickwilliams-eventbrite why are we assuming extensions will always be less than 9 (aka be represented by 1 digit)?